uk.co.piggz.amazfish.json

```diff
@@ -0,0 +1,120 @@
+{
+  "app-id": "uk.co.piggz.amazfish",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-23.08",
+  "sdk": "org.kde.Sdk",
+  "command": "run-amazfish.sh",
+  "rename-icon": "harbour-amazfish-ui",
+  "rename-desktop-file": "harbour-amazfish-ui.desktop",
+  "rename-appdata-file": "harbour-amazfish-ui.appdata.xml",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--system-talk-name=org.bluez",
+    "--talk-name=org.mpris.MediaPlayer2.*",
+    "--talk-name=io.github.rinigus.PureMaps",
```

Comment:
Why does it need access to another app's bus name?

Reply:
bluez is used for communication with smartwatch
mpris is used for control of media player from smartwatch
puremaps is used for obtaining navigation instruction and showing on smartwatch